app/scripts/lib/auth-errors.js

```diff
@@ -225,6 +225,27 @@ function (_, Errors) {
     SIGNUP_DISABLED_BY_RELIER: {
       errno: 1030,
       message: t('Signup has been disabled')
+    },
+    ALL_NUMBERS_LETTERS: {
+      errno: 1031,
```

Comment:
All of these new errno's need to be added to the [client-metrics doc](https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md#signup)